ios/Sources/Core/Context.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Context.swift
+//  tauri-plugin-admob
+//
+//  Created by Seto Elkahfi on 2025-11-19.
+//
```

Comment:
Git blame is enough to know who edited a file. Please remove this type of comment